test/ibmq/test_proxies.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests for the IBMQClient proxy support."""
+
+import subprocess
+
+from requests.exceptions import ProxyError
+
+from qiskit.providers.ibmq.api_v2 import IBMQClient
+from qiskit.providers.ibmq.api_v2.exceptions import RequestsApiError
+from qiskit.test import QiskitTestCase, requires_qe_access
+
+from ..decorators import requires_new_api_auth
+
+ADDRESS = '127.0.0.1'
+PORT = '8080'
+
+
+class TestProxies(QiskitTestCase):
+    """Tests for proxy capabilities."""
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        # Launch the mock server.
+        cls.proxy_process = subprocess.Popen([
+            'pproxy', '-v', '-i', 'http://{}:{}'.format(ADDRESS, PORT)
+        ], stdout=subprocess.PIPE)
+
+    @classmethod
+    def tearDownClass(cls):
+        super().tearDownClass()
+
+        # Close the mock server.
+        if cls.proxy_process.returncode is None:
+            cls.proxy_process.terminate()
+
+    @requires_qe_access
+    @requires_new_api_auth
+    def test_proxies(self, qe_token, qe_url):
+        """Should reach the proxy."""
+        request = 'http auth.quantum-computing.ibm.com:443'
```

Comment:
Can we find a slightly more descriptive name for this variable (`request` is a bit overloaded), and more importantly make its content based on `qe_url`?